.github/workflows/pr-title.yaml

```diff
@@ -0,0 +1,31 @@
+name: PR Title Linting
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
```

Comment:
is this needed?

Reply:
Yeah it triggers when the workflow will run. So maybe synchronize is not needed but I see no harm keeping it.

Reply:
I guess my question was: what is the default types value? Is it smaller or bigger than what is here? I would assume the default would just trigger on all of these 3 types, plus maybe more?

Reply:
Defaults are opened, synchronize, and reopened. https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request. So we would like edited as well